website/docs/reference/dbt-jinja-functions/local_md5.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "local_md5"
+id: "local_md5"
+---
+<Changelog>New in 1.4.0</Changelog>
```

Comment:
We don't use these `<Changelog>` entries any more. This system predated the introduction of versioned pages + `VersionBlock`, with the intent of replacing it entirely.

Instead, we should add this page to the [`exports.versionedPages` list](https://github.com/dbt-labs/docs.getdbt.com/blob/current/website/dbt-versions.js#L28), with `"firstVersion": "1.4"`. That requires having added v1.4 to the `exports.versions` list, as described in https://github.com/dbt-labs/docs.getdbt.com/issues/2427. (It looks like that change is missing from https://github.com/dbt-labs/docs.getdbt.com/pull/2646.)

Reply:
Great catch. I'll get the export.versions list updated as part of #2534